src/cpu/core_dyn_x86/cache.h

```diff
@@ -149,7 +151,7 @@ class CodePageHandler : public PageHandler {
 			}
 			memset(invalidation_map,0,4096);
 		}
-		(*(Bit16u*)&invalidation_map[addr])+=0x101;
```

Comment:
PPC: Fix needs to be aligned and native endian (not swapped to little-endian like the `host_*` functions do).